packages/smooth_app/lib/pages/product/product_incomplete_card.dart

```diff
@@ -21,6 +21,10 @@ class ProductIncompleteCard extends StatelessWidget {
   final bool isLoggedInMandatory;
 
   static bool isProductIncomplete(final Product product) {
+    if (product.productType != null &&
```

Comment:
the name doesn't completely reflect what's actually going on

Reply:
> the name doesn't completely reflect what's actually going on

You're right. The thing is that we display the "hey your product is incomplete" message only when the product is incomplete, for food products, therefore `isProductIncomplete` made sense.
I've just added an "if not food, consider as complete" test.
I can rename the method to `isIncompleteFoodProduct`: would that be OK with you?